packages/api-client/src/views/Request/RequestSection/RequestCodeExample.vue

```diff
@@ -233,7 +233,6 @@ const customCodeContent = computed(() => {
             @update:modelValue="selectClient">
             <ScalarButton
               class="text-c-2 hover:text-c-1 flex h-full w-fit gap-1.5 px-1.25 py-0.75 font-normal"
-              fullWidth
```

Comment:
Already has `w-fit` in the classname so this was being overridden anyway

(I swear more than half the places we're applying this prop we override it in the class)